src/controllers/authentication.controller.js

```diff
@@ -24,7 +26,7 @@ const {
   findUserByEmailOrUsername,
   findUserEmailIfExist,
   updateUserPassword,
-  updateIsVerified
+  updateIsVerifiedOrDisableNotification
```

Comment:
Could you rename this function to maybe `updateIsVerifiedOrUpdateNotification` since the part of notifications deals with both disable and enable?